test/unity/toolkits/drawing_classifier/test_dc_serialization.cxx

```diff
@@ -24,11 +21,15 @@ namespace {
 class drawing_classifier_mock
     : public turi::drawing_classifier::drawing_classifier {
  public:
-  std::unique_ptr<neural_net::model_spec> my_init_model() const {
+  std::unique_ptr<neural_net::model_spec> my_get_model_spec() const {
     return init_model();
   }
```

Comment:
Instead of defining `my_get_model_spec` (we generally avoid first-person nomenclature), could we just expose the base class function like this:
```
public:
  using drawing_classifier::init_model;
```
(see also https://en.cppreference.com/w/cpp/language/using_declaration)

Reply:
init_model is protected.

Reply:
okay, I should say goodbye to `my`, which I have a lot of fun with.